platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -74,7 +74,7 @@ namespace platoon_strategic_ihp
      * 
```

Comment:
Are all the notes here still relevant?  That is, do these actions still need to be done?  If so, please add a "TODO" tag so that it can easily be identified in a search.  And confirm for me that these will be done soon, right?